crm_schemas.go

```diff
@@ -0,0 +1,64 @@
+package hubspot
+
+import "fmt"
+
+const (
+	crmSchemasPath = "schemas"
+)
+
+// CrmSchemas is an interface of CRM crm/v3/schemas interface..
+// Reference: https://developers.hubspot.com/docs/api/crm/crm-custom-objects
+type CrmSchemasService interface {
+	List() (map[string]interface{}, error)
+	Create(map[string]interface{}) (map[string]interface{}, error)
+	Get(string) (map[string]interface{}, error)
+	Delete(string, bool) error
+	Update(string, map[string]interface{}) (map[string]interface{}, error)
+}
+
+// CrmSchemasServiceOp handles communication with the CRM schemas endpoint.
+type CrmSchemasServiceOp struct {
+	client         *Client
+	crmSchemasPath string
+}
+
+var _ CrmSchemasService = (*CrmSchemasServiceOp)(nil)
+
+func (s *CrmSchemasServiceOp) List() (map[string]interface{}, error) {
```

Comment:
I think that it is better to define a response type if possible.
(Same PR comment as CrmProperties)

Reply:
Made basically the same changes as I did in crm_properties.

Update/Create now can take an interface{} instead of map[string]interface{} and everything returns concrete objects (`CrmSchema`)